UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.IntegrationTests/RadioNavigationalWarning/Admin/RadioNavigationalWarningAdminTest.cs

```diff
@@ -0,0 +1,174 @@
+﻿using FakeItEasy;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+using System;
+using System.Threading.Tasks;
+using UKHO.MaritimeSafetyInformation.Common.Configuration;
+using UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning;
+using UKHO.MaritimeSafetyInformation.Web.Controllers;
+using UKHO.MaritimeSafetyInformation.Web.Services;
+using UKHO.MaritimeSafetyInformation.Web.Services.Interfaces;
+
+namespace UKHO.MaritimeSafetyInformation.IntegrationTests.RadioNavigationalWarning.Admin
+{
+    [TestFixture]
+    public class RadioNavigationalWarningAdminTest : BaseRNWTest
+    {
+        public ILogger<RadioNavigationalWarningsAdminController> _fakeLogger;
+        private IRNWRepository _rnwRepository;
+        private RNWService _rnwService;
+
+        private RadioNavigationalWarningsAdminController _controller;
+
+        [OneTimeSetUp]
+        public async Task OneTimeSetUp()
+        {
+            await SeedRadioNavigationalWarnings(GetFakeRadioNavigationalWarnings());
+            await SeedWarningType(GetFakeWarningTypes());
+        }
+
+        [SetUp]
+        public void Setup()
+        {
+            _fakeLogger = A.Fake<ILogger<RadioNavigationalWarningsAdminController>>();
+            _rnwRepository = new RNWRepository(_fakeContext);
+            _rnwService = new RNWService(_rnwRepository, _fakeRadioNavigationalWarningConfiguration, _fakeLoggerRnwService);
+
+            _controller = new RadioNavigationalWarningsAdminController(_fakeHttpContextAccessor, _fakeLogger, _rnwService);
+        }
+
+        [Test]
+        public async Task WhenCallIndex_ThenReturnListAsync()
+        {
+            _fakeRadioNavigationalWarningConfiguration.Value.AdminListRecordPerPage = 20;
+            IActionResult result = await _controller.Index(1, null, null);
+            RadioNavigationalWarningsAdminFilter adminListFilter = (RadioNavigationalWarningsAdminFilter)((ViewResult)result).Model;
+
+            Assert.AreEqual(8, adminListFilter.RadioNavigationalWarningsAdminList.Count);
+            Assert.AreEqual(1, adminListFilter.PageCount);
+            Assert.AreEqual(0, adminListFilter.SrNo);
+            Assert.AreEqual(WarningTypes.NAVAREA_1, adminListFilter.CurrentPageIndex);
```

Comment:
It seems Index value is changed as WarningTypes.NAVAREA_1 which is incorrect.
```suggestion
            Assert.AreEqual(1, adminListFilter.CurrentPageIndex);
```

Reply:
Changes Done